.githooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# full paths from the repo root separated by newlines
+MUST_NOT_CHANGE='js/canvas-to-blob.min.js
+js/jquery.cloudinary.js
+js/jquery.fileupload.js
+js/jquery.fileupload-image.js
+js/jquery.fileupload-process.js
+js/jquery.fileupload-validate.js
+js/jquery.iframe-transport.js
+js/jquery.ui.widget.js
+js/load-image.all.min.js
+js/README.md
+'
+
+if git rev-parse --verify HEAD >/dev/null 2>&1
```

Comment:
Can't we just run the jasmine tests we already have?
Doesn't this test exactly the same thing but in a slightly different way?

Reply:
yes but it prevents a commit, while the test does not. it doesn't hurt

Reply:
I'm convinced :) 

If however at some point we'll want to change that, well need to update two places